doc/2/plugins/plugin-context/accessors/realtime/index.md

```diff
@@ -0,0 +1,117 @@
+---
+code: true
+type: page
+title: registerSubscription
+---
+
+# `realtime`
+
+Adds or removes [real-time subscriptions](/core/2/guides/essentials/real-time) from the backend.
+
+## `subscribe`
+
+Registers a new realtime subscription on behalf of a client. The subscription works exactly like the one created by the [`realtime:subscribe` API endpoint](/api/controllers/realtime/subscribe/index.md). The notifications will be sent to the connection identified by the `Request` object passed to the method.
```

Comment:
Nitpicking: we try to use the term "action" and not method or endpoint which can be confusing (the first with a method in the code and the second because it's mainly related to HTTP)
```suggestion
Registers a new realtime subscription on behalf of a client. The subscription works exactly like the one created by the [realtime:subscribe](/api/controllers/realtime/subscribe/index.md) API action. The notifications will be sent to the connection identified by the `Request` object passed to the method.
```